client/my-sites/customer-home/components/home-survey/index.jsx

```diff
@@ -0,0 +1,55 @@
+import { useIsEnglishLocale } from '@automattic/i18n-utils';
+import { __ } from '@wordpress/i18n';
```

Comment:
Instead of importing `__` directly I prefer to use `useI18n()` or `useTranslate` so that we're sure that the translation mechanism is ready. But it's not a big deal.